### DISCUSSION TITLE
mybatis-plus version is too old.

### DISCUSSION THREAD

**Discussion by YiuTerran** [apache/dolphinscheduler#8162] [General]
Mybatis-plus v3.2.0 using mybatis 3.5.2. 

There are high level security problems for this version. See [CVE-2020-26945](https://nvd.nist.gov/vuln/detail/CVE-2020-26945)

**Comment by YiuTerran** [comment]
Does anyone care about this? Our company had decided not to use dolphin-scheduler for this reason.

**Reply by lenboo** [reply]
Thanks very much for reminding!
We will fix it in the new version.

**Reply by YiuTerran** [reply]
There is no fix even now...
